os/kernel/Kconfig

```diff
@@ -966,7 +966,6 @@ config IDLETHREAD_STACKSIZE
 config USERMAIN_STACKSIZE
 	int "Main thread stack size"
 	default 2048
-	depends on !BINARY_MANAGER
```

Comment:
With binary manager support on binary separation, We don't need to execute user app because binary manager loads user binary application.
But now binary manager works in flat build also, user app should in flat build.
Please refer to line 318 in os_bringup.c.